transform/models/staging/stg_gharchive.sql

```diff
@@ -1,5 +1,5 @@
 select
-   replace(type,'Event','') as event_type, --remove 'Event' suffix
+   replace(type,'Event','') as LOWER(event_type), 
```

Comment:
Style: Since you went with uppercase SQL keywords, let's stay consistent with SELECT and REPLACE here.